### PR TITLE
Build contracts to `/artifacts` dir only and use it as source for all pipelines.

### PR DESCRIPTION
**Pull request by deuszx**

### CODE REVIEW

File: amm/scripts/prepare_rust_wrappers.sh [hunk 1]
```diff
@@ -6,50 +6,26 @@
...
-        echo "Copying $c.wasm"
-        cp $SCRIPT_DIR/../../target/ink/$c/$c.wasm $SCRIPT_DIR/../drink-tests/resources/$c.wasm;
-    done
-    echo "Copying wrapped_zero.wasm" &&
-    cp $SCRIPT_DIR/../../test-contracts/wrapped-azero/target/ink/wrapped_azero.wasm $SCRIPT_DIR/../drink-tests/resources/wrapped_azero.wasm ;
-    echo "Copying psp22.wasm" &&
-    cp $SCRIPT_DIR/../../test-contracts/psp22/target/ink/psp22.wasm $SCRIPT_DIR/../drink-tests/resources/psp22.wasm ;
-}
 
 function run() {
```
Comment (h4nsu) [inline amm/scripts/prepare_rust_wrappers.sh:27]: This is probably not needed anymore

**Comment by deuszx** [comment]
@h4nsu I removed the e2e tests now. There was a lot of problems w/ its dependencies:
* old aleph-client
* old ink primitives
and some other mix of "forced" dependencies in `Cargo.lock` to make it work.

I checked all of the tests it had an we're already covering them with drink! tests.

File: amm/Makefile [hunk 2]
```diff
@@ -31,15 +32,8 @@ check-amm: check-drink-tests ## Runs cargo (contract) check on AMM contracts.
...
-.PHONY: build-test-contracts
-build-test-contracts: ## Builds contracts used in e2e-tests
-	@for d in $(TEST_PATHS); do \
-		echo "Building $$d contract" ; \
-		cd $$d && cargo contract build --quiet --release --features "contract" && cd $(SCRIPT_DIR) ; \
-	done
-
 .PHONY: build-all
-build-all: build-amm build-test-contracts ## Builds all contracts.
+build-all: build-amm ## Builds all contracts.
```
Comment (h4nsu) [inline amm/Makefile:36]: Pedantic nit: How about removing `build-amm` phony and moving its contents to `build-all`? They are kinda synonyms now...

**Review by h4nsu — APPROVED** [review]

File: amm/scripts/prepare_rust_wrappers.sh (hunk 1 above)
Comment (deuszx) [reply amm/scripts/prepare_rust_wrappers.sh]: Removed.

File: amm/Makefile (hunk 2 above)
Comment (deuszx) [reply amm/Makefile]: Removed.

**Pull request merged**